src/test/resources/good_csvs/sales.csv

```diff
@@ -1,5 +1,5 @@
-"Project Name","Item ARK","Parent ARK","Item Status ID","Item Status","Object Type","File Name","Item Sequence","Duplicate","Delete in Title","Rights.copyrightStatus","Name.interviewee","Language","Type.typeOfResource","Type.genre","Name.repository","Subject","Name.interviewer","Name.subject","Description.note","Date.creation","Date.normalized","Format.extent","Title","Alt ID.local","Description.abstract"
```

Comment:
Not a big deal (not a blocker), but I like explicit quotes in CSVs. I realize they're not necessary here though.

Viva la verbosidad!